src/providers/app.provider.ts

```diff
@@ -17,6 +19,10 @@ export const init = (name: string, controllers: any[], origin?: string[]) => {
 
   const app = express();
   const logger = Logger(name);
+  const corsOptions = {
+    origin: 'http://localhost:3000',
```

Comment:
This doesn't have to be hardcoded here, line 15's signature accept an array of origin. Which can be dynamically passed in the `*.server.ts` for Admin and Non-Admin